src/test/scala/io/phdata/retirementage/filters/HdfsStorageTest.scala

```diff
@@ -185,4 +186,20 @@ class HdfsStorageTest extends FunSuite with SparkTestBase {
         .getOrElse(throw new Exception("properties not found")))
   }
 
+  test("read avro table") {
+    val table    = DatedTable("tableA", "avro", "date", 1, None, None, None)
+    val database = Database("default", Seq(table))
+    val schema   = StructType(StructField("date", LongType, false) :: Nil)
+    val data     = TestObjects.smallDatasetSeconds
+
+    val df       = createDataFrame(data, schema)
+    val location = s"target/test-tables/${database.name}.${table.name}"
+
+    df.write.mode(SaveMode.Overwrite).avro(location)
+
+    val newdf = spark.read.avro(location)
+    
+    fail()
```

Comment:
For this test you can call the `persistFrame` method. After that just assert that the directory it creates exists.

Reply:
Then you can do the same for parquet.